style/style.css

```diff
@@ -0,0 +1,11 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+h1 {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: #6a47fb;
+}

```

Comment:
- [ ] It is a good practice to use class selector rather than element selector to add style to your file. Kinldy use the class name from your `index.html` to add styling to its content.